src/Extensions/Compatability/Logger/Category.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.Extensions.Compatability.Logger
+{
+	/// <summary>
+	/// Logging category
+	/// </summary>
+	public struct Category : IEquatable<Category>
+	{
+		/// <summary>
+		/// Category
+		/// </summary>
+		/// <param name="name">Category name</param>
+		public Category(string name) => Name = name;
+
+
+		/// <summary>
+		/// Name
+		/// </summary>
+		public string Name { get; }
+
+
+		/// <summary>
+		/// Indicates whether the current object is equal to another object of the same type
+		/// </summary>
+		/// <param name="other">An object to compare with this object</param>
```

Comment:
Missing `<returns>` here and throughout this class

Reply:
There is no support of categories for new logger and I don't think there is any point of improving comments for copied compatability class, at least for now, beside that return type for `Equals` method is too verbose from my point of view, it's hard to misunderstand it.